bokehjs/src/lib/models/glyphs/line.ts

```diff
@@ -43,17 +44,30 @@ export class LineView extends XYGlyphView {
       const sx_i = sx[i]
       const sy_i = sy[i]
 
+      if (nonselection && !move && iprev != null && i-iprev > 1 && isFinite(sx[iprev+1] + sy[iprev+1]))
```

Comment:
is `iprev != null && i-iprev > 1` actually the same as `gap(i)` ?

Reply:
It is the same as `gap(i)`. But if we change it to use `gap(i)` here then this `if` statement no longer includes the `iprev != null` check so we need to the use the TypeScript non-null assertion exclamation mark after `iprev` 4 times in this and the next line.

I marginally prefer to keep it as it is as I think it is clearer to non-specialists in TypeScript. But I will change it if you prefer.